content/02.delete-me.md

```diff
@@ -1,15 +1,165 @@
+
+---
+
+A long paragraph of text.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+Another long paragraph.
+Donec massa sapien faucibus et molestie ac feugiat sed lectus.
+Est sit amet facilisis magna etiam.
+Ornare arcu dui vivamus arcu felis bibendum.
+Magna fermentum iaculis eu non diam.
+Vehicula ipsum a arcu cursus vitae.
+Ante metus dictum at tempor commodo ullamcorper a.
+Vitae tortor condimentum lacinia quis vel eros.
+Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet.
+Facilisis sed odio morbi quis.
+Enim ut sem viverra aliquet eget sit amet tellus.
+
+*Italic* _text_
+
+**Bold** __text__
+
+Combined *italics and __bold__*
+
+~~Strikethrough~~
+
+Line break without starting a new paragraph by putting  
+two spaces at end of line.
+
+Putting each sentence on its own line has numerous benefits with regard to [editing](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) and [version control](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+- List item
+- List item
+- List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+unicode superscript™
+
+unicode subscriptₓ
+
+**Links and Citations**
+
+Bare URL link: <https://manubot.org>
+
+[Link with text](https://www.manubot.org)
+
+[Link with hover text](https://www.manubot.org "Manubot Homepage")
+
+[Link styled as a button](https://www.manubot.org "Manubot Homepage"){.button}
+
+[Link by reference][manubot homepage]
+
+[Manubot Homepage]: https://www.manubot.org
+
+Citation via DOI: [@doi:10.15363/thinklab.4]
+
+Citation by tag: [@tag:deep-review]
+
+Manual URL citation: [@url:https://greenelab.github.io/meta-review]
+
+**Figures and Tables**
+
+Figure @fig:square-image
+
+Figure @fig:wide-image
+
+Figure @fig:tall-image
+
+Figure @fig:vector-image
+
+Table @tbl:bowling-scores
+
+**Quotes and Code**
+
+> Quoted text
+
+> Quoted block of text  
+>
+> Two roads diverged in a wood, and I—  
+> I took the one less traveled by,  
+> And that has made all the difference.  
+
+Code `in the middle` of normal text, aka `inline code`
+
+```c++
+// code block with C++ syntax highlighting
+int main() {
+    // a string too long to fit within page
+    cout << "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness";
+    return 0;
+}
+```
+
+```
+// code block with no syntax highlighting
+int count = 0;
+function add() {
+	count++;
+}
```

Comment:
Should we change this code block to:

```

Code block with no syntax highlighting:

``
Exporting HTML manuscript
Exporting DOCX manuscript
Exporting PDF manuscript
``
```

(but with three backticks).

The motivation is that we don't want to give users bad habits of omitting syntax highlighting when there should be syntax highlighting.